doc/user/distribution.md

```diff
@@ -12,13 +12,11 @@ and with or without Graal are also made available on the
 Currently, these distributions are limited to Linux as we do not know a way to
 get OpenJDK 8 binaries for other platforms on which to build the JVMCI support.
 
-To use such a distribution, you need to unpack it and
-`source` the included script `truffleruby/setup_env` to setup the right
-environment ($PATH, gems config, etc):
+To use such a distribution, unpack it and add `truffleruby/bin` to your `$PATH`:
 
 ```bash
 $ tar xf truffleruby-....tar.gz
-$ source truffleruby/setup_env
+$ export PATH="$(pwd)/truffleruby/bin:$PATH"
```

Comment:
Just a tiny nit (and I'm not sure it's worth bothering about) but `PATH="$PWD/truffleruby/bin:$PATH"` would save the command substitution from being needed and should be portable.

Reply:
Right, this should work in most shells and CI configs, thanks for the suggestion :)